prime-dotnet-webapi/ViewModels/SiteRegistration/_SiteMappingProfile.cs

```diff
@@ -13,6 +13,12 @@ public SiteMappingProfile()
                 .ForMember(dest => dest.HasAcceptedAgreement, opt => opt.MapFrom(src => src.Agreements.Any(a => a.AcceptedDate.HasValue)))
                 .ForMember(dest => dest.HasSubmittedSite, opt => opt.MapFrom(src => src.Sites.Any(s => s.SubmittedDate.HasValue)))
                 .ForMember(dest => dest.Sites, opt => opt.MapFrom(src => src.Sites.Where(s => (careSettingCode == null || s.CareSettingCode == careSettingCode) && s.ArchivedDate == null)));
+            CreateMap<Organization, OrganizationAdminListViewModel>()
+                .ForMember(dest => dest.SigningAuthorityName, opt => opt.MapFrom(src => $"{src.SigningAuthority.FirstName} {src.SigningAuthority.LastName}"))
+                .ForMember(dest => dest.SiteId, opt => opt.MapFrom(src => src.Sites.FirstOrDefault(s => s.PEC != null).Id))
+                .ForMember(dest => dest.HasClaim, opt => opt.MapFrom(src => src.Claims.Any()))
+                .ForMember(dest => dest.ValidSiteCount, opt => opt.MapFrom(src => src.Sites.Count(s => s.PEC != null)))
```

Comment:
I think more accurate to test `CareSettingCode`

Reply:
Updated as suggested